packages/steppp/src/index.ts

```diff
@@ -55,37 +46,18 @@ function Steppp(element: HTMLElement, options: any = defaultOptions): Instance {
     return buildAnimation(args);
   };
 
-  const getFrames = (
-    direction: Direction
-  ): {
-    oldStepFrames: Frame[];
-    newStepFrames: Frame[];
-  } => {
-    const backward = isMovingBackward(direction);
-    const { enter, exit } = animationFrames;
-
-    return {
-      oldStepFrames: backward ? flip(exit) : exit,
-      newStepFrames: backward ? flip(enter) : enter,
-    };
```

Comment:
This `flip` crap is the crux of the change. I already know the "enter" and "exit" frames using the `computeAnimationFrames()` already in this file. Doing some weird flip here is just overcomplicating things.